### PR TITLE
[pfsense] bugfix for OPNsense syslog ingest pipeline

### PR DESCRIPTION
**Pull request by fabiogermann**
## What does this PR do?
Some pattern changes are necessary for proper funciton with OPNsense:

An example of such a log looks like the following:
```
<134>1 2022-05-15T11:54:18-04:00 opn001.test.intra.net filterlog 76549 - [meta sequenceId="7299"] 13,,,02f4bab031b57d1e30553ce08e0ec131,igb1,match,block,in,4,0x0,,128,56110,0,none,17,udp,78,192.168.1.8,192.168.2.255,137,137,58
```
```
<13>1 2022-05-15T11:39:04-04:00 opn001.test.intra.net /flowd_aggregate.py 73841 - [meta sequenceId="3653"] vacuum done
```
- Addition of `PF_SEQ_ID` in `default.yml`: the grok expression to handle `[meta sequenceId="7299"] ` was missing for all logs
- Change from `INT` to `WORD` in `firewall.yml`: the string `02f4bab031b57d1e30553ce08e0ec131` represents the rule id which had to be adapted
- Change from `WORD` to `WORD_WITH_DOT` in `default.yml`: process name can also contain a dot like in `/flowd_aggregate.py`

## Checklist

- [ ] I have reviewed [tips for building integrations](https://github.com/elastic/integrations/blob/main/docs/tips_for_building_integrations.md) and this pull request is aligned with them.
- [x] I have verified that all data streams collect metrics or logs.
- [x] I have added an entry to my package's `changelog.yml` file.
- [x] I have verified that Kibana version constraints are current according to [guidelines](https://github.com/elastic/elastic-package/blob/master/docs/howto/stack_version_support.md#when-to-update-the-condition).

## Author's Checklist

- [ ] Make sure changes are compatible with pfSense
- [ ] Make sure changes are compatible with OPNsense

## How to test this PR locally

Fresh installation of OPNsense (latest version) logs over TCP to Elastic Agent with pfSense policy configured.

## Related issues

n/a

## Screenshots

n/a

### CODE REVIEW

**Comment by fabiogermann** [comment]
@legoguy1000 As the integration is originally from you I wnated to check back. I have found some errors in v1.0.0 in relation to OPNsense (22.1.7_1) and this PR should resolve them.

**Review by efd6 — COMMENTED** [review]
Thank you for sending this fix.

Would you be able to provide test case log lines that will test the change; the current state already passes, so these are not adequate to test the change.

**Comment by P1llus** [comment]
Sorry for the delay @fabiogermann , Are you able to resolve the current conflicts so I can do a last pass before merging it?

File: packages/pfsense/data_stream/log/_dev/test/pipeline/test-opensense.log [hunk 1]
```diff
@@ -2,4 +2,6 @@
 <134>Dec 31 22:02:55 firewall.opnsense.net filterlog[15307]: 6,,,1000000105,igb1.27,match,block,in,6,0x00,0xf6279,1,UDP,17,32,fe80::208:9bff:fef3:652b,ff02::1:2,546,547,32
 <190>Dec 31 22:06:16 firewall.opnsense.net dhcpd[40152]: DHCPDISCOVER from 4c:55:41:a0:fa:99 via eth0.60
 <30>Dec 31 22:06:33 firewall.opnsense.net unbound[75499]: [26931:0] info: 192.168.1.1 api.opensubtitles.org. A IN
-<134>1 2022-06-09T14:44:11-06:00 OPNsense.example.com filterlog 76404 - [meta sequenceId="1"] 124,,,fae559338f65e11c53669fc3642c93c2,ixl1_vlan70,match,pass,out,4,0x0,,63,5687,0,DF,6,tcp,60,192.168.1.1,10.10.10.10,40370,80,0,S,3364871769,,65535,,mss;sackOK;TS;nop;wscale
\ No newline at end of file
+<134>1 2022-06-09T14:44:11-06:00 OPNsense.example.com filterlog 76404 - [meta sequenceId="1"] 124,,,fae559338f65e11c53669fc3642c93c2,ixl1_vlan70,match,pass,out,4,0x0,,63,5687,0,DF,6,tcp,60,192.168.1.1,10.10.10.10,40370,80,0,S,3364871769,,65535,,mss;sackOK;TS;nop;wscale
```
Comment (P1llus) [inline packages/pfsense/data_stream/log/_dev/test/pipeline/test-opensense.log:5]: Is it only github, or does this line end with space/newline?

**Review by P1llus — APPROVED** [review]
All seems good, after conflicts are fixed we could merge. @efd6 Will fix those I believe.

File: packages/pfsense/data_stream/log/_dev/test/pipeline/test-opensense.log (hunk 1 above)
Comment (efd6) [reply packages/pfsense/data_stream/log/_dev/test/pipeline/test-opensense.log:5]: It's a newline-only locally.

**Pull request merged**